src/main/java/seedu/address/model/person/Person.java

```diff
@@ -240,4 +240,18 @@ public int compareTo(Person o) {
         return this.getName().toString().compareToIgnoreCase(o.getName().toString());
     }
 
+    /**
+     * Used in debugging to print out the relationshipList of this person
+     */
+    public void printRelationshipList() {
```

Comment:
Consider using the native logging mechanisms for this?

Reply:
emmm if i m not wrong the logging is for all the steps the app went through? If thats the case i d rather use this coz the logs wont be unnecessarily long. And since this method is only for debugging purposes, the message printed out using System.out.print is of color white instead of red, which makes it easier for me to distinguish from other information logs.

Reply:
I was thinking of having this being printed on the `FINE` log level actually, would that be more suitable? See [here](https://github.com/CS2103AUG2017-F10-B1/main/blob/master/docs/DeveloperGuide.adoc#logging).